docs/concepts/sensor-models/sensor-models.md

```diff
@@ -0,0 +1,88 @@
+
+Sensor models that support full transformation from the image plane to the BCBF coordinate system fall into two general classes -- rigorous models and rational functional models.  Rigorous sensor models function by taking ephemeris information (position, orientation, and velocity) with respect to some reference frame and utilize mathematical models to spatialize the image location onto the target body [(Laura et al., 2020)](https://doi.org/10.1029/2019EA000713).  Rational functional models approximate this information using rational polynomial coefficients without exposing the interior and exterior orientation of the instrument.  Within this document (and the ASC sensor model ecosystem), we focus on rigorous models due to the availability of interior and exterior orientation information.
+
+
+## The Community Sensor Model
+
+Because the only requirement of a sensor model is that it models a sensor's geometric information, the design and implementation of a sensor model largely remains open-ended.  Of particular importance is the sensor's interface, i.e. what information is required to create the model and what information is produced by the model.  Without a concrete interface specification, each sensor model may require and provide its own set of information, which makes it incredibly difficult to facilitate interoperability between sensor models. In order to address these concerns about interoperability, the US defense and intelligence community established the [Community Sensor Model (CSM) working group](https://gwg.nga.mil/gwg/focus-groups/Community_Sensor_Model_Working_Group_(CSMWG).html) , which is responsible for developing and maintaining a set of standards upon which sensor models can be implemented.  The resulting CSM standard describes an application programming interface (API) for multiple sensor types.
+
+Alone, a CSM is of little practical use. The standard defines an interface but does not define where necessary sensor metadata should be sourced, the format of said metadata, or the implementation of the photogrammetric algorithms used by the sensor. Therefore, a CSM exists within a broader ecosystem. First, some external capability or method must be used to generate image support data (ISD) that can be used to parameterize a CSM. Then, the CSM can parse and use the ISD to extract the parameters (interior and exterior orientation) necessary to perform photogrammetric operations (as defined by the CSM API). Finally, a SET is likely used to orchestrate the pieces of the CSM ecosystem and exploits said pieces in order to use the sensor for some desired operation [(Laura et al., 2020)](https://doi.org/10.1029/2019EA000713)
```

Comment:
Consider: '...some external capability or method must be used to generate the inputs (intrinsics and extrinsic) to the sensor model. The CSM calls these inputs image support data.'

Also, the aha moment I am having here is that we assume the reader knows that the sensor model is this abstract thing that you can re-use over and over again with different inputs, but we never tell them that! A valid question would be 'is it sensor model or sensor models?' and 'is the sensor model reusable?'.